e2e/specs/forms-dashboard.spec.ts

```diff
@@ -16,54 +11,53 @@ test.beforeEach(async ({ api }) => {
   await addFormResources(api, valueReference, form.uuid);
 });
 
-test("Filter forms based on publish status", async ({ page }) => {
+test('Filter forms based on publish status', async ({ page }) => {
   const formBuilderPage = new FormBuilderPage(page);
 
-  await test.step("When I visit the form builder", async () => {
+  await test.step('When I visit the form builder', async () => {
     await formBuilderPage.gotoFormBuilder();
   });
 
   // Test the filter functionality
-  await test.step("Then I click the publish filter dropdown", async () => {
+  await test.step('Then I click the publish filter dropdown', async () => {
     await page
-      .getByRole("combobox", {
-        name: "Filter by publish status: All Open menu",
+      .getByRole('combobox', {
+        name: 'Filter by publish status: All Open menu',
```

Comment:
I don't think this test could've ever succeeded. The DOM element with the role `combobox` is the button which only has the text "All".

Reply:
And it works!